### PR TITLE
Javacli98 add --ignore-naming-conflicts flag for Bulk Put  

### PR DESCRIPTION
**Pull request by berkeleysquare**
Also remove --force flag from Delete Job. It is now always set to force in the sdk (artifact of 3.2 merge?)

### CODE REVIEW

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java [hunk 1]
```diff
@@ -168,6 +170,8 @@ public Arguments(final String[] args) throws BadArgumentException, ParseExceptio
         noFollowSymlinks.setLongOpt("no-follow-symlinks");
         final Option followSymLinks = new Option(null, false, "Set to follow symlinks");
         followSymLinks.setLongOpt("follow-symlinks");
+        final Option ignoreNamingConflicts = new Option(null, false, "Set true to ignore existing files of the same name during a bulk put");
```
Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java]: This should say `same name and size`

File: ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java [hunk 2]
```diff
@@ -968,6 +969,40 @@ public void putBulk() throws Exception {
...
+    public void putBulkWithIgnoreConflicts() throws Exception {
+        final Arguments args = new Arguments(new String[]{"ds3_java_cli", "-e", "localhost:8080", "-k", "key!",
+                "-a", "access", "-c", "put_bulk", "-b", "bucketName", "-d", "dir", "--ignore-naming-conflicts"});
+        final Ds3ClientHelpers helpers = mock(Ds3ClientHelpers.class);
+        final Ds3ClientHelpers.Job mockedPutJob = mock(Ds3ClientHelpers.Job.class);
+        final FileUtils mockedFileUtils = mock(FileUtils.class);
+
+        final Path p1 = Paths.get("obj1.txt");
+        final Path p2 = Paths.get("obj2.txt");
+        final ImmutableList<Path> retPath = ImmutableList.copyOf(Lists.newArrayList(p1, p2));
```
Comment (rpmoore) [inline ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java]: The right assignment of this should be:

```java
ImmutableList.of(p1, p2);
```

File: ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java [hunk 3]
```diff
@@ -968,6 +969,40 @@ public void putBulk() throws Exception {
...
+
+        final UUID jobId = UUID.randomUUID();
+        when(mockedPutJob.getJobId()).thenReturn(jobId);
+        final Iterable<Ds3Object> retObj = Lists.newArrayList(new Ds3Object("obj1.txt", 1245), new Ds3Object("obj2.txt", 12345));
```
Comment (rpmoore) [inline ds3_java_cli/src/test/java/com/spectralogic/ds3cli/Ds3Cli_Test.java]: This should be updated to use an ImmutableList

File: ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteJob.java [hunk 4]
```diff
@@ -42,17 +40,14 @@ public CliCommand init(final Arguments args) throws Exception {
...
-        }
         return this;
     }
 
     @Override
     public DefaultResult call() throws Exception {
-        final CancelJobSpectraS3Request request = new CancelJobSpectraS3Request(id).withForce(this.force);
-        getClient().cancelJobSpectraS3(request);
-        return new DefaultResult("SUCCESS: Deleted job '"+ this.id.toString() +"'");
+        // Force is always on in CancelJobSpectraS3Request
```
Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteJob.java]: In this case the force flag should be used to determine if CancelJob is called, or ifTruncateJob is called.

Comment (berkeleysquare) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteJob.java]: Aha, very good. I just did this to make it build; .withForce() was suddenly missing. No objection to fixing it here under a different issue?

Comment (rpmoore) [inline ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/DeleteJob.java]: That should be fine.

**Pull request merged**